index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+    <title>web page portfolio</title>
+</head>
+<body>
+    <header>
+        <nav class="container">
+            <ul>
+                <li><h2 class="baritem">Welcome</h2></li>          
+                <li><a href="/" rel="noopener"><img src="./Union.svg" alt="menu" ></li></a>
+            </ul>
```

Comment:
- **[OPTIONAL]:** Great job implementing the requested changes from the previous reviewer! 💯 However, having an `<ul>` tag here doesn't make much sense because you're not describing a list of elements. Please consider changing it! 🛠️